test/test_internal.jl

```diff
@@ -0,0 +1,58 @@
+import UUIDs
+
+"""Recursively validates that the object and fields have UUIDs."""
+function validate_uuids(obj::T) where T
+    if !(obj isa PowerSystemType)
+        return true
+    end
+
+    result = true
+    if !(PowerSystems.get_uuid(obj) isa Base.UUID)
+        result = false
+        @error "object does not have a UUID" obj
```

Comment:
This `@error` macro does not throw an error, just logs an error.

Reply:
My intention was to log every violation instead of failing on the first one.

Reply:
Ditto.